src/vs/nls.ts

```diff
@@ -202,7 +202,16 @@ export function localize2(data: ILocalizeInfo | string, message: string, ...args
 	return {
 		value: original,
 		original
-	};
+	} as ILocalizedString;
+}
+
+/**
+ * Create a `ILocalizedString` object that represents an unlocalized string.
+ *
+ * This lets you explicitly pass in unlocalized strings to places that normally expect localized strings.
+ */
+export function unlocalized2(str: string): ILocalizedString {
```

Comment:
This change needs to be done in the nls.build file as well. That's likely what caused the build failures